roles/oneagent/tests/conftest.py

```diff
@@ -13,12 +13,12 @@
 from ansible.runner import AnsibleRunner
 from command.platform_command_wrapper import PlatformCommandWrapper
 from util.common_utils import prepare_test_dirs
-from util.constants.common_constants import (
-    COMPONENT_TEST_BASE,
-    INSTALLERS_DIRECTORY,
-    LOG_DIRECTORY,
-    SERVER_DIRECTORY,
-    TEST_DIRECTORY,
+from constants import (
+    TEST_RUN_DIR_PATH,
+    WORK_DIR_INSTALLERS_PATH,
```

Comment:
```suggestion
    INSTALLERS_DIR_PATH,
```
Preceding a variable name with the name of a parent path creates a vicious cycle. Same with `WORK_DIR_LOGS_PATH`, `WORK_DIR_SERVER_PATH`.